client/account-recovery/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import controller from './controller';
+
+export default function() {
```

Comment:
Even if this is only client-side, I might suggest considering the isomorphic render router as a point of future compatibility if SSR is ever desired:

https://github.com/Automattic/wp-calypso/blob/master/docs/isomorphic-routing.md

As an example of a browser-only section using it:

https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/types/index.js